site/_includes/layouts/base.njk

```diff
@@ -92,8 +94,6 @@
     </div>
     {% include 'partials/cookie-banner.njk' %}
 
-    {% block chromescripts %}{% endblock %}
-
```

Comment:
Reverse this change.

Reply:
You need to put that line back, since you're now not moving that to the head since you reversed the line above.

Also looks like things have changed so you have merge conflicts. It might be easiest to just close this PR and open a new one, from `main` with just the `defer` change?

Reply:
Ok 🤝